go/vt/schemadiff/table.go

```diff
@@ -458,6 +464,17 @@ func (c *CreateTableEntity) normalizeColumnOptions() {
 	}
 }
 
+func (c *CreateTableEntity) normalizeIndexOptions() {
```

Comment:
I supposed this function isn't strictly related to the performance improvements?

Reply:
It is definitely relevant to the changes. This is because the `sqlparser` comparison functions compare the direct AST and there we have these values stored in a non normalized way before this change. This means that specific index options with different casing were seen as a change instead of the same.

Before the `CanonicalString` usage was hiding that, since that would normalize to upcase but if we use the `sqlparser` functions, we need this normalization to be consistent.

It's similar to how we normalize other things though like the type for a column.